tests/testGeneralShapeletPsfApproxPlugins.py

```diff
@@ -34,6 +34,7 @@
 import lsst.log.utils
 import lsst.meas.modelfit
 import lsst.meas.base
+import lsst.meas.algorithms
```

Comment:
`lsst.meas.modelfit` previously imported code that imported `lsst.meas.algorithms`, and since that just puts the `algorithms` subpackage into the existing `lsst.meas` package due to our unusual import hooks, the need for `lsst.meas.algorithms` here was disguised.